app/test/frontend/golden/flutter_packages_index_page2.html

```diff
@@ -99,7 +99,9 @@
   </thead>
   <tbody>
       <tr>
-        <td><img src="/static/img/flutter-logo-32x32.png" class="tablecol-icons-img" alt="Flutter package" title="Flutter package"/>
+        <td>  <div style="min-width: px">
```

Comment:
As a TODO, once we have a proper CSS story, it might make sense to move that into the `tablecol-icons-img` css class.

also: Isn't there a number missing before `px`? Maybe avoid emitting it alltogether if there is no pixel with given.

Reply:
missing px: good catch, that is for the tables that are still using detectTypes. Fixed their template for now.

style vs css: Agree and noted, but without a TODO. I hope the new UI templates should already handle it, and will revisit on migration.